.buildkite/pipeline.yml

```diff
@@ -194,6 +194,15 @@ steps:
           slurm_ntasks: 32
           slurm_mem_per_cpu: 16G
 
+      - label: "Unthreaded AMIP FINE"
+        key: "unthreaded_amip_fine"
+        command: "julia --color=yes --project=experiments/AMIP/moist_mpi_earth/ experiments/AMIP/moist_mpi_earth/coupler_driver_modular.jl --run_name target_amip_n1_shortrun --enable_threading false --coupled true --surface_scheme monin_obukhov --moist equil --vert_diff true --rad gray --z_elem 50 --dz_top 3000 --dz_bottom 30 --h_elem 16 --kappa_4 1e16 --z_stretch false --rayleigh_sponge true --alpha_rayleigh_uh 0 --alpha_rayleigh_w 10 --dt_cpl 150 --dt 150secs --dt_rad 1hours --FLOAT_TYPE Float64 --energy_check false --mode_name amip --t_end 0.01days --dt_save_to_sol 1days --mono_surface false --precip_model 0M"
```

Comment:
Yes, good idea! I can change the MPI test (since its parameters were breaking for the long run anyway) and we can revamp the ci pipeline to include more options when we do physics testing. .